### PR TITLE
Adding warning message about openjdk 11 issue on redhat

### PR DESCRIPTION
**Pull request by EFM-Bobby**
Pushing this first and will correct formatting after I see the generated pages.

### CODE REVIEW

**Comment by EFM-Bobby** [comment]
@kelpoole This look ok to you? The two notes are added on these pages:
https://deploy-preview-4520--edb-docs-staging.netlify.app/docs/efm/latest/installing/prerequisites/
https://deploy-preview-4520--edb-docs-staging.netlify.app/docs/efm/latest/13_troubleshooting/#confirming-the-openjdk-version

**Comment by EFM-Bobby** [comment]
@drothery-edb Hi, can you give this a look and go ahead and merge it when you get the chance? If we need to change something later that's fine, but I'd like to get this merged in before have another PR for efm 4.8 changes.

File: product_docs/docs/efm/4/13_troubleshooting.mdx [hunk 1]
```diff
@@ -47,3 +47,9 @@ openjdk version "1.8.0_191"
 OpenJDK Runtime Environment (build 1.8.0_191-b12)
 OpenJDK 64-Bit Server VM (build 25.191-b12, mixed mode)
 ```
+!!! Note
+    There is a temporary issue with OpenJDK version 11 on RHEL and its derivatives. When starting Failover Manager, you may see an error like the following:
+
+    `java.lang.Error: java.io.FileNotFoundException: /usr/lib/jvm/java-11-openjdk-11.0.20.0.8-2.el8.x86_64/lib/tzdb.dat (No such file or directory)`
+
+    If so, the workaround is to manually install the missing package, e.g. `dnf install tzdata-java`
```
Comment (djw-m) [inline product_docs/docs/efm/4/13_troubleshooting.mdx:55]:
```suggestion
    If so, the workaround is to manually install the missing package using the command `sudo dnf install tzdata-java`
```

File: product_docs/docs/efm/4/installing/prerequisites.mdx [hunk 2]
```diff
@@ -17,6 +17,13 @@ Before configuring a Failover Manager cluster, you must satisfy the prerequisite
 
 Before using Failover Manager, you must first install Java (version 1.8 or later). Failover Manager is tested with OpenJDK, and we strongly recommend installing that version of Java. [Installation instructions for Java](https://openjdk.java.net/install/) are platform specific.
 
+!!! Note
+    There is a temporary issue with OpenJDK version 11 on RHEL and its derivatives. When starting Failover Manager, you may see an error like the following:
+
+    `java.lang.Error: java.io.FileNotFoundException: /usr/lib/jvm/java-11-openjdk-11.0.20.0.8-2.el8.x86_64/lib/tzdb.dat (No such file or directory)`
+
+    If so, the workaround is to manually install the missing package, e.g. `dnf install tzdata-java`
```
Comment (djw-m) [inline product_docs/docs/efm/4/installing/prerequisites.mdx:25]:
```suggestion
    If so, the workaround is to manually install the missing package using the command `sudo dnf install tzdata-java`
```

Comment (EFM-Bobby) [reply product_docs/docs/efm/4/installing/prerequisites.mdx]: Centos 7 users would have to install dnf to run that exact command, but I'm ok with it. :)

Comment (djw-m) [reply product_docs/docs/efm/4/installing/prerequisites.mdx]: Could we add ("or `sudo yum install tzdata-java`")

Comment (EFM-Bobby) [reply product_docs/docs/efm/4/installing/prerequisites.mdx]: I don't think we need to. That's why I only had the one command as an example; if someone is still using centos 7 and doesn't know dnf=yum I'd be surprised. And I might be the only person still using centos 7.

Comment (djw-m) [reply product_docs/docs/efm/4/installing/prerequisites.mdx]: Ok, just wanted to make sure there wasn't underlying doubts.

**Review by djw-m — APPROVED** [review]
LGTM

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**